packages/bpk-stylesheets/index.scss

```diff
@@ -16,65 +16,77 @@
  * limitations under the License.
  */
 
-@import '~normalize.css';
-@import '../bpk-mixins/index.scss';
+@layer reset, theme, utility;
 
-// stylelint-disable-next-line selector-max-type
-html {
-  font-size: $bpk-font-size-root;
-  box-sizing: border-box;
+@layer reset {
+  @import '~normalize.css';
 }
 
-* {
-  box-sizing: inherit;
+@layer theme {
+  @import '../bpk-mixins/index.scss';
 
-  &::before,
-  &::after {
-    box-sizing: inherit;
+  // stylelint-disable-next-line selector-max-type
+  html {
+    font-size: $bpk-font-size-root;
+    box-sizing: border-box;
   }
-}
 
-/* stylelint-disable selector-class-pattern */
-body {
-  color: $bpk-font-color-base;
-  font-family: $bpk-font-family-base;
-  font-size: $bpk-font-size-base;
-  line-height: 1.3rem; /* stylelint-disable-line scale-unlimited/declaration-strict-value */
+  * {
+    box-sizing: inherit;
 
-  :global(&.scaffold-font-size) {
-    font-size: 13px; /* stylelint-disable-line unit-disallowed-list, scale-unlimited/declaration-strict-value */
+    &::before,
+    &::after {
+      box-sizing: inherit;
+    }
   }
 
-  :global(&.enable-font-smoothing) {
-    -webkit-font-smoothing: antialiased;
+  /* stylelint-disable selector-class-pattern */
+  body {
+    color: $bpk-font-color-base;
+    font-family: $bpk-font-family-base;
+    font-size: $bpk-font-size-base;
+    line-height: 1.3rem; /* stylelint-disable-line scale-unlimited/declaration-strict-value */
+
+    :global(&.scaffold-font-size) {
+      font-size: 13px; /* stylelint-disable-line unit-disallowed-list, scale-unlimited/declaration-strict-value */
+    }
+
+    :global(&.enable-font-smoothing) {
+      -webkit-font-smoothing: antialiased;
+    }
   }
 }
 
-:focus-visible {
-  @include bpk-focus-indicator;
-}
+@layer utility {
+  /* stylelint-disable-next-line no-duplicate-at-import-rules, no-invalid-position-at-import-rule */
```

Comment:
Had to disable two lint rules here, mixins needs imported twice, each for the two layers.
This could be solved by combining theme and utility into one layer

Reply:
I was going to say everything in this file should come under one layer that I would see as `base` as this is core across everything and acts as the base the system then builds on